aspnetcore/mvc/advanced/app-parts.md

```diff
@@ -91,16 +90,39 @@ The `GenericController` class:
 
 [!code-csharp[](./app-parts/sample/AppPartsSample/GenericController.cs?highlight=5-6)]
 
-The result, when a matching route is requested:
+The result when requesting `http://localhost:28431/Sprocket`:
 
-![Example output from the sample app reads, 'Hello from a generic Sprocket controller.'](app-parts/_static/generic-controller.png)
+```text
+Hello from a generic Sprocket controller.
```

Comment:
The output was shown in a screenshot, so I've pulled it in as text instead. I dunno if this is really needed, but I didn't want to _just_ remove the image.